cluster/images/etcd/Makefile

```diff
@@ -192,6 +192,7 @@ endif
 	# And build the image
 	docker buildx build \
 		--pull \
+		 --sbom=false --provenance=false \
```

Comment:
nit:
```suggestion
		--sbom=false \
		--provenance=false \
```

Reply:
I check some other examples:
- `--provenance` is necessary
- I removed the sbom part.